doc/source/conf.py

```diff
@@ -217,8 +217,9 @@
 # Intersphinx configuration
 # -----------------------------------------------------------------------------
 intersphinx_mapping = {
-        'http://docs.python.org/dev': None,
-        'https://docs.scipy.org/doc/numpy': None,
+        'python': ('http://docs.python.org/dev', None),
+        'numpy': ('https://docs.scipy.org/doc/numpy', None),
```

Comment:
Oh and @WarrenWeckesser these changes are just moving from the old way of mapping, which has been [deprecated since Sphinx 1.0](http://www.sphinx-doc.org/en/stable/ext/intersphinx.html#confval-intersphinx_mapping).

Reply:
OK, thanks for the link.

Reply:
this looks fine to me